pkg/operator/ceph/object/bucket/controller.go

```diff
@@ -129,6 +131,8 @@ func (r *ReconcileBucket) reconcile(request reconcile.Request) (reconcile.Result
 		return reconcile.Result{}, nil
 	}
 
+	r.cephClusterSpec = &cephCluster.Spec
```

Comment:
With the other suggested change to avoid changing LoadClusterInfo(), do we not need this change anymore? Seems like all this controller needed to know is if the cephcluster CR exists, and doesn't need to actually retrieve it.